pkg/aggregator/aggregator.go

```diff
@@ -168,6 +170,8 @@ func init() {
 	aggregatorExpvars.Set("DogstatsdContexts", &aggregatorDogstatsdContexts)
 	aggregatorExpvars.Set("EventPlatformEvents", &aggregatorEventPlatformEvents)
 	aggregatorExpvars.Set("EventPlatformEventsErrors", &aggregatorEventPlatformEventsErrors)
+	aggregatorExpvars.Set("ContainerLifecycleEvents", &aggregatorContainerLifecycleEvents)
+	aggregatorExpvars.Set("ContainerLifecycleEventErrors", &aggregatorContainerLifecycleEventErrors)
```

Comment:
nit: for consistency you may want to go with `ContainerLifecycleEventsErrors` (added a `s` to `Events`)